### ISSUE TITLE
Some HTTP detects as Apple

### ISSUE THREAD

**Issue by zar1n** [ntop/nDPI#155]
Hello,

Link to flow dump file - https://www.dropbox.com/s/oh8iwl6auxafclz/gesti-mnogo.net.pcap?dl=0
URL - http://gestimnogo.net (nsfw)

```
Detected protocols:
    Apple                packets: 1068          bytes: 145956        flows: 6            


Protocol statistics:
    Acceptable                  145956 bytes

    1   TCP 109.110.59.8:16147 <-> 104.28.10.17:80 [proto: 140/Apple][166 pkts/25517 bytes]
    2   TCP 109.110.59.8:16151 <-> 104.28.10.17:80 [proto: 140/Apple][113 pkts/16800 bytes]
    3   TCP 109.110.59.8:16153 <-> 104.28.10.17:80 [proto: 140/Apple][115 pkts/20849 bytes]
    4   TCP 109.110.59.8:15432 <-> 104.28.10.17:80 [proto: 140/Apple][114 pkts/28115 bytes]
    5   TCP 109.110.59.8:16148 <-> 104.28.10.17:80 [proto: 140/Apple][487 pkts/36686 bytes]
    6   TCP 109.110.59.8:16150 <-> 104.28.10.17:80 [proto: 140/Apple][73 pkts/17989 bytes]
```

And several URLs without flow dump files:
http://www.best-prava.eu/orenburg.html
http://www.originals-diploms.com/ 
http://kalyanmir.ru/category/tabak/ 
http://best-prava.eu/price.html
http://best-prava.eu/index.html
http://cash-forum.biz/topic/4418-prodazha-udostoverenii-mvdskprokuratury-avto/
http://slotsdoc.ru/igrovye-avtomaty/book-of-ra.htm

And many other...

Used nDPI version 1.7.0 from sourcefordge.

Thank you.

**Comment by lucaderi** [comment]
Fixed in 1.7.1

Using nDPI (1.7.1-dev-299-20374b5) [1 thread(s)]
Reading packets from pcap file /Users/deri/Downloads/gesti-mnogo.net.pcap...
Running thread 0...

nDPI Memory statistics:
    nDPI Memory (once):      105.91 KB  
    Flow Memory (per flow):  1.88 KB  
    Actual Memory:           1.91 MB  
    Peak Memory:             1.91 MB      

Traffic statistics:
    Ethernet bytes:        171588        (includes ethernet CRC/IFC/trailer)
    Discarded bytes:       0  
    IP packets:            1068          of 1068 packets total
    IP bytes:              145956        (avg pkt size 136 bytes)
    Unique flows:          6  
    TCP Packets:           1068  
    UDP Packets:           0  
    VLAN Packets:          0  
    MPLS Packets:          0  
    PPPoE Packets:         0  
    Fragmented Packets:    0  
    Max Packet size:       1062  
    Packet Len < 64:       960  
    Packet Len 64-128:     0  
    Packet Len 128-256:    0  
    Packet Len 256-1024:   106  
    Packet Len 1024-1500:  2  
    Packet Len > 1500:     0  
    nDPI throughput:       371.74 K pps / 455.66 Mb/sec
    Traffic throughput:    112.87 pps / 141.68 Kb/sec
    Traffic duration:      9.462 sec
    Guessed flow protos:   6            

Detected protocols:
    Cloudflare           packets: 1068          bytes: 145956        flows: 6            

Protocol statistics:
    Acceptable                  145956 bytes

```
1   TCP 109.110.59.8:16147 <-> 104.28.10.17:80 [proto: 7.220/HTTP.Cloudflare][166 pkts/25517 bytes][Host: gestimnogo.net]
2   TCP 109.110.59.8:16151 <-> 104.28.10.17:80 [proto: 7.220/HTTP.Cloudflare][113 pkts/16800 bytes][Host: gestimnogo.net]
3   TCP 109.110.59.8:16153 <-> 104.28.10.17:80 [proto: 7.220/HTTP.Cloudflare][115 pkts/20849 bytes][Host: gestimnogo.net]
4   TCP 109.110.59.8:15432 <-> 104.28.10.17:80 [proto: 7.220/HTTP.Cloudflare][114 pkts/28115 bytes][Host: gestimnogo.net]
5   TCP 109.110.59.8:16148 <-> 104.28.10.17:80 [proto: 7.220/HTTP.Cloudflare][487 pkts/36686 bytes][Host: gestimnogo.net]
6   TCP 109.110.59.8:16150 <-> 104.28.10.17:80 [proto: 7.220/HTTP.Cloudflare][73 pkts/17989 bytes][Host: gestimnogo.net]
```

**Comment by kYroL01** [comment]
@zar1n just to make you better understand why is detected as CloudFlare, see http://whois.domaintools.com/gestimnogo.net
